Topics/Tech_Stacks/CSS.md

```diff
@@ -3,6 +3,7 @@
 ## Table of contents
 ### [Introduction](#introduction-1)
 ### [CSS Cascade](#css-cascade-1)
+### [CSS Selector Specificity](#css-selector-specificity-1)
```

Comment:
It may also be nice to add extra hyperlinks to the other subsections such as Specificity Hierarchy, How is specificity calculated?, Troubleshooting, Additional Resources:

Reply:
I think I disagree with adding hyperlinks to the table of contents, since it is meant to be a high-level list of concepts, and to keep consistent with other topics, which aren't as extensively written about. Should there be more contributions in the future, I'll be sure to add some subpoints to the table.